nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -187,6 +182,7 @@
             .description("Max wait time for connection to remote service.")
             .required(true)
             .defaultValue("5 secs")
+            .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
```

Comment:
Is there a reason this property descriptor will only support variable registry values rather than flowfile attributes as done in `PROP_READ_TIMEOUT`?

Reply:
READ_TIMEOUT and CONNECTION_TIMEOUT are initialized before onTrigger method.

I think CONNECTION_TIMEOUT might be static for every request so i use only VARIABLE_REGISTRY for this property because of i mentioned above, but READ_TIMEOUT should be different for every single request so I use FLOWFILE_REGISTRY for this property.

I assign READ_TIMEOUT value via interceptor(TimeoutInterceptor) on onTrigger method because okHttpClient does not allow to change timeout parameter.

I don't mark as resolved. If i have a wrong information please send comment.

Thank you.